config/application.rb

```diff
@@ -1,6 +1,7 @@
 require File.expand_path('../boot', __FILE__)
 
 require 'rails/all'
+require 'elasticsearch/rails/instrumentation'
```

Comment:
I've done in config/application:

```ruby
    # Run as soon as the application constant inherits from Rails::Application.
    # Thus, initializers required in it will be as if they were in this file
    # and will be evaluated before Rails initializes.
    config.before_configuration do
      require Rails.root.join("config/initializers/00_application_config")
    end
```